flink-runtime/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaRpcService.java

```diff
@@ -108,9 +111,11 @@
 
 	private volatile boolean stopped;
 
-	public AkkaRpcService(final ActorSystem actorSystem, final Time timeout) {
+	public AkkaRpcService(final ActorSystem actorSystem, final Configuration configuration) {
```

Comment:
Why did you pass the `Configuration` here? I think it would be better to revert this change.

Reply:
Sorry, I forgot to refactor this code segment : 

```
if (actorSystem.settings().config().hasPath(MAXIMUM_FRAME_SIZE_PATH)) {
			maximumFramesize = actorSystem.settings().config().getBytes(MAXIMUM_FRAME_SIZE_PATH);
		} else {
			// only local communication
			maximumFramesize = Long.MAX_VALUE;
		}
```

to use

```
configuration.getString(AkkaOptions.FRAMESIZE);
```

(this suggestion from @zentol)

If I would do this, I think it's better to replace scattered config items into one Configuration object so that we can encapsulate change?

Reply:
Yes let's introduce an `AkkaRpcServiceConfiguration`.